docker/docker-compose-embedded.yaml

```diff
@@ -32,10 +32,11 @@ services:
       - 5001:5001
 
   lotus:
-    image: textile/lotus-devnet:sha-7ed2544
+    image: textile/lotus-devnet:sha-1e166c9
     ports:
       - 7777:7777
     environment:
       - TEXLOTUSDEVNET_SPEED=1500
+      - TEXLOTUSDEVNET_BIGSECTORS=true
```

Comment:
The devnet now allos to specify if you want to use "big sectors", so supporting 512Mib sectors.
By default uses 2Kb sectors, which is what is running for the CI tests... tried 512Mib but it's too heavy for parallelization.